test/Microsoft.Azure.Devices.Edge.Test/helpers/Context.cs

```diff
@@ -109,6 +109,7 @@ IEnumerable<Registry> GetAndValidateRegistries()
             this.TestTimeout = TimeSpan.FromMinutes(context.GetValue("testTimeoutMinutes", 5));
             this.Verbose = context.GetValue<bool>("verbose");
             this.ParentHostname = Option.Maybe(Get("parentHostname"));
+            this.BlobSasUrl = Option.Maybe(Get("E2E_BLOB_STORE_SAS"));
```

Comment:
Dang...one more thing. `Get("E2E_BLOB_STORE_SAS")` probably won't find anything. The `ConfigurationBuilder` pulls in all env vars that start with `E2E_` on [line 20](https://github.com/Azure/iotedge/blob/72a1fb3b3fbe8502ce017694dfb8bb4e3a0018f0/test/Microsoft.Azure.Devices.Edge.Test/helpers/Context.cs#L20), then you refer to them _without_ the prefix. So it should be:
```csharp
this.BlobSasUrl = Option.Maybe(Get("BLOB_STORE_SAS"));
```